client/src/pages/Apply/Apply.js

```diff
@@ -95,13 +96,16 @@ const Apply = ({ navigation }) => {
     if (acceptTerms === false) {
       validateErrors.acceptTerms = 'Terms are required'
     }
+    if (followUp === false) {
+      validateErrors.followUp = 'Follow up is required'
+    }
     setErrors(validateErrors)
```

Comment:
It looks like the state variable `errors` is no longer being used. Hence, all code related to the state variable `errors` should be removed because we don't want unnecessary code in the repo.

Also, it looks like `areRequiredFieldsFilled` achieves the same functionality as `validate()`. So, better off to just delete `validate()` and replace its use with `areRequiredFieldsFilled`.